example/generated-src/objc/TextSort-Bridging-Header.h

```diff
@@ -0,0 +1,7 @@
+// AUTOGENERATED FILE - DO NOT MODIFY!
```

Comment:
Would best-practice suggest also generating a .m file which includes this?  I noticed that in the old diff.
I'm fine either way, though if the new arg causes 2 files to be generated, it should take a base name to which both extensions can be added, rather than taking the full name of one file and removing its extension to name another file.